tests/spec/core/linter-rules/warn-broken-refs-spec.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+describe("Core Linter Rule - 'warn-broken-refs'", () => {
+  const ruleName = "warn-broken-refs";
```

Comment:
Nit: I think we should inline this. Doesn't seem to add much by having it as a variable.